packages/echo-core/src/__tests__/services/graphUtils.test.ts

```diff
@@ -30,7 +31,7 @@ const globalConsoleMethod = global.console;
 const globalFetchMethod = window.fetch;
 beforeAll(() => {
     // removes console log in test run
-    global.console = ({ log: jest.fn(), error: jest.fn() } as unknown) as Console;
+    global.console = { log: jest.fn(), error: jest.fn() } as any;
```

Comment:
we should try to avoid any. I think it's better to have it typed as before?

Reply:
The parentheses caused some lint errors via prettier:
![image](https://user-images.githubusercontent.com/13872908/135216660-2516e36d-2410-43bf-bce7-883b1672111c.png)
So I had to remove them, and the type casting as well. n___n;

Generally I challenge the idea to apply strict typing in unit tests, as they can make testing quite complex and time-consuming in more complex scenarios.

But I think I can refactor these a bit to be able to keep the typing and also satisfying the prettier.